gulpfile.js

```diff
@@ -54,5 +55,25 @@ gulp.task('fonts', function() {
         .pipe(gulp.dest(path.join(output_directory, 'fonts')));
 });
 
+// Extracts translatable strings using angular-gettext and saves them in file
+// openslides/locale/angular-gettext/template-en.pot.
+gulp.task('pot', function () {
+    return gulp.src(['openslides/core/static/templates/*.html',
+                     'openslides/users/static/templates/users/*.html',
+                     'openslides/users/static/js/users/*.js'])
```

Comment:
Can you use 'openslides/*/static/templates/*/*.html' and 'openslides/*/static/js/*/*.js' ? Maybe 'openslides/*/static/**.js' and 'openslides/*/static/**.html' also works.

What about plugins?